ui/app/templates/components/dashboard/replication-card.hbs

```diff
@@ -0,0 +1,100 @@
+{{!
+  Copyright (c) HashiCorp, Inc.
+  SPDX-License-Identifier: BUSL-1.1
+~}}
+
+<div class="is-flex-between">
+  <h3 class="title is-4 has-bottom-margin-xxs" data-test-client-count-title>
+    Replication
+  </h3>
+
+  <LinkTo class="is-no-underline" @route="vault.cluster.replication.index">
+    Details
+  </LinkTo>
+</div>
+
+{{! check if dr replication and performance replication exists }}
+{{#if (or @replication.dr.clusterId @replication.performance.clusterId)}}
+  <hr class="has-background-gray-100" />
+  {{! check if user has access to both perf replication and dr replication }}
+  {{#if (and @version.hasPerfReplication @version.hasDRReplication)}}
+    <div
+      class="is-grid grid-2-columns grid-gap-2 has-top-margin-m has-bottom-margin-xs grid-align-items-start"
+      data-test-dr-perf-replication
+    >
+      <Dashboard::ReplicationStateText
+        @title="DR primary"
+        @name="dr"
+        @state={{@replication.dr.state}}
+        @clusterStates={{cluster-states @replication.dr.state}}
+      />
+      <div>
+        {{#if @replication.performance.clusterId}}
+          <Dashboard::ReplicationStateText
+            @title="Perf primary"
+            @name="performance"
+            @state={{@replication.performance.state}}
+            @clusterStates={{cluster-states @replication.performance.state}}
+          />
+        {{else}}
+          <Dashboard::ReplicationStateText @title="Perf primary" @name="performance" />
+        {{/if}}
```

Comment:
Do we need both of these here? It seems like you mostly handle this logic in the component. Just wondering if it could be the following (not sure how it affects the helper 💭 )
```html
<Dashboard::ReplicationStateText
 @title="Perf primary"
 @name="performance"
 @state={{if @replication.performance.clusterId @replication.performance.state "not set up"}}
 @clusterStates={{cluster-states @replication.performance.state}}
/>
```